src/chrome/content/rules/Sci-Hub.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+	Broken chain:
+		sci-hub.name
+		*.sci-hub.name
+		sci-hub.hk
+		*.sci-hub.hk
+		sci-hub.mn
+		*.sci-hub.mn
+		*.sci-hub.tv
+		*.sci-hub.tw
```

Comment:
tw and tv seems to be fixed.

Reply:
I already added `^` for these 2 domains. `www` is still misconfigured though.